apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -45,20 +45,21 @@ import { useModel } from "@/lib/hooks/use-model";
 import { useThread } from "@/lib/hooks/use-thread";
 import { cn, scrollToBottomOfElement } from "@/lib/utils";
-import { createThread, getThread, saveNewMessage } from "@/services/hasura";
+import { createThread, deleteThread, getThread, saveNewMessage } from "@/services/hasura";
 import type {
   AiClientType,
-  ChatLoadingState,
   ChatProps,
-  CleanPromptResult,
+  CleanPromptResult
 } from "@/types/types";
+import { WordWareFlowPaths } from "@/types/wordware-flows.types";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)